src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -329,9 +328,6 @@ MTR_AVAILABLE(ios(16.1), macos(13.0), watchos(9.1), tvos(16.1))
 /**
  * Download log of the desired type from the device.
  *
- * Note: The consumer of this API should move the file that the url points to or open it for reading before the
- * completion handler returns. Otherwise, the file will be deleted, and the data will be lost.
```

Comment:
Why is this being removed?  Is this no longer true?  Why does the MTRBaseDevice version not have this?